swarm/network/stream/delivery.go

```diff
@@ -138,6 +131,7 @@ type RetrieveRequestMsg struct {
 	SkipCheck bool
 }
 
+// TODO: Fix context SNAFU
```

Comment:
What is SNAFU?

Reply:
https://www.urbandictionary.com/define.php?term=SNAFU

Reply:
But I removed the comment, because context is handled properly now, it was just a leftover comment